src/pages/playlist/PlaylistVideos.jsx

```diff
@@ -0,0 +1,74 @@
+import { Fragment } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import axios from "axios";
+import { usePlaylist } from "../../context/playlist/playlist-context";
+import { VideoCard } from "../../components/video-card/VideoCard";
+import { REMOVE_FROM_PLAYLISTS } from "../../shared/types";
+import { useAuth } from "../../context/authentication/auth-context";
+import { toast } from "react-toastify";
+
+export const PlaylistVideo = () => {
+  const { playlistId } = useParams();
+  const navigate = useNavigate();
+  const {
+    playlistState: { playlists },
+    playlistDispatch,
+  } = usePlaylist();
+
+  const {
+    authState: { token },
+  } = useAuth();
+
+  const activePlaylist = playlists.find((item) => item._id === playlistId);
+
+  const deleteVideoFromPlaylistHandler = async (e) => {
+    try {
+      const response = await axios.delete(
+        `/api/user/playlists/${activePlaylist._id}/${e.target.id}`,
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      if (response.status === 200) {
+        playlistDispatch({
+          type: REMOVE_FROM_PLAYLISTS,
+          payload: response.data.playlist,
+        });
+        toast.info(`removed successfully`);
+      }
+    } catch (error) {}
```

Comment:
Same here ,if app fails to do certain action then user should know why it failed so use toast to display information.